src/hypothesis/extra/datetime.py

```diff
@@ -70,6 +70,40 @@ def do_draw(self, data):
                 pass
 
 
+class TimedeltaStrategy(SearchStrategy):
+
+    def __init__(self, min_days=None, max_days=None):
```

Comment:
I think having day-range here is not good enough. How to make timedeltas for minute-wide range? Hour range? I think here better would be to have some min_value, max_value and units or mimic timedelta constructor interface to accept all those minutes, hours, days arguments (actually only one of these at once) as two value tuple of min and max value.

Day-wide timedelta is quite too broad.